.vscode/settings.json

```diff
@@ -99,7 +99,11 @@
         "codecvt": "cpp",
         "ranges": "cpp",
         "scoped_allocator": "cpp",
-        "typeindex": "cpp"
+        "typeindex": "cpp",
```

Comment:
Are these intentional or is this a vestigial change? I noticed that vscode likes to cram stuff here all the time no reason seemingly.

Reply:
That's VSCode just adding things itself, and me assuming it knows what it's doing while modifying its own settings file. Do you think it's a problem or otherwise better to avoid committing these changes?

Reply:
https://github.com/microsoft/vscode-cpptools/issues/722#issuecomment-301136398 I had no idea how this worked either.  I think we should just add this file to the gitignore, it just adds visual noise to PRs :man_shrugging:

Reply:
ok, that works for me. I can imagine we might want to actually have shared workspace settings at some point, but we don't have any meaningful ones at the moment so might as well cut down on the noise.